clients/web/src/stylesheets/layout/layout.styl

```diff
@@ -19,6 +19,7 @@
   top 65px
   right 10px
   width 320px
+  z-index 11000
```

Comment:
I assume this is just to make sure it is above progress messages. 

Would it be better to have this set computationally?  That is, `$headerZ+2` instead of `11000` and `$headerZ+1` instead of `10000` elsewhere in the file (or define a different range for such values)?